AccountingService/Services/JwtTokenService.cs

```diff
@@ -6,25 +6,31 @@
 using Microsoft.Extensions.Configuration.Json;
 using Microsoft.Extensions.Configuration;
 using System.Text;
+using AccountingService.Interface;
 
 namespace AccountingService.Services
 {
     public class JwtTokenService
     {
         private readonly IConfiguration _configuration;
-        public JwtTokenService(IConfiguration configuration)
+        private readonly ITimeService _timeService;
+
+
+        public JwtTokenService(IConfiguration configuration, ITimeService timeService)
         {
             _configuration = configuration;
+            _timeService = timeService;
         }
         public Token GetToken(List<Claim> claims)
         {
             var authSigningKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(_configuration["JwtConfig:Key"]));
-            var expiryInMinutes = Convert.ToInt32(15);
+            var expiryInMinutes = _timeService.Minutes;
+
             var token = new JwtSecurityToken(
                 claims: claims,
                issuer: _configuration["JwtConfig:Issuer"],
                 audience: _configuration["JwtConfig:Audience"],
-                expires: DateTime.Now.AddMinutes(expiryInMinutes),
+                 expires: DateTime.Now.AddMinutes(expiryInMinutes),
```

Comment:
You exposed DateTime.Now from the new TimeService but you are not using it here. Why?